templates/fluent-resource/en-US/tools.ftl

```diff
@@ -1,4 +1,52 @@
-tools-write-ide-prose = Whether you prefer working with code from the command line, or using
-        rich graphical editors, there’s a Rust integration available for your
-        editor of choice. Or you can build your own using the
+# index.hbs
+tools-editor-support-heading = First-class editor support
+tools-write-ide-description = Whether you prefer working with code from the
```

Comment:
update the name of this id in `xx-AU` too, please

Reply:
Sure thing! Do you mind if I rename it (in both files) to `tools-editor-support-description` for consistency?

Reply:
go for it